app/src/main/java/edu/calvin/cs262/teamc/MyMatchesActivity.java

```diff
@@ -97,39 +104,41 @@ public void onResponse(String response) {
         }, new Response.ErrorListener() {
             @Override
             public void onErrorResponse(VolleyError error) {
-             //   Toast.makeText(MyMatchesActivity.this, "Failed", Toast.LENGTH_SHORT).show();
-
             }
         }){
 
             @Override
             protected Map<String, String> getParams() throws AuthFailureError {
                 Map<String, String> postMap = new HashMap<>();
                 postMap.put("Content-Type","application/json");
-                //..... Add as many key value pairs in the map as necessary for your request
                 return postMap;
             }
         };
-        //make the request to your server as indicated in your request url
         Volley.newRequestQueue(getApplicationContext()).add(stringRequest);
 
+        // Proc each ListView item to open the activity of the corresponding match
         lv.setOnItemClickListener(new AdapterView.OnItemClickListener() {
             @Override
             public void onItemClick(AdapterView<?> parent, View view, int position, long id)
             {
                 Intent i = new Intent(getBaseContext(), MatchActivity.class);
+
                 i.putExtra("loginID",getIntent().getStringExtra("loginID"));
                 i.putExtra("name", matches.get(position).getName());
-                Log.e("imgID", matches.get(position).imgSrc);
                 i.putExtra("imgId", matches.get(position).imgSrc);
+
                 startActivity(i);
             }
         });
 
```

Comment:
Extra line

Reply:
I assume you mean 129, which is supposed to be there.